pkg/cmd/cli/cmd/status.go

```diff
@@ -160,7 +161,7 @@ func (o StatusOptions) RunStatus() error {
 		if err != nil {
 			return err
 		}
-		data, err := dot.Marshal(g, o.namespace, "", "  ", false)
+		data, err := dot.Marshal(g, quote(o.namespace), "", "  ", false)
```

Comment:
make sure all the places we call dot.Marshal use this... I see it in describe/chaindescriber.go as well, which means we might need to move/export the quote helper func